src/cartesian/ErrorBar.tsx

```diff
@@ -43,12 +44,30 @@ interface ErrorBarProps extends InternalErrorBarProps {
    * Only accepts a value of "x" or "y" and makes the error bars lie in that direction.
    */
   direction?: 'x' | 'y';
+  isAnimationActive?: boolean;
+  animationBegin?: number;
+  animationDuration?: number;
+  animationEasing?: AnimationTiming;
```

Comment:
can we add tests for each of the added properties?

1. animation can be disabled
2. animation starts after delay
3. animationDuration can be modified
4. animationEasing can be modified